src/main/scala/org/apache/spark/sql/delta/sources/DeltaSource.scala

```diff
@@ -371,4 +381,30 @@ case class DeltaSource(
       case other => throw new UnsupportedOperationException(s"Unknown ReadLimit: $other")
     }
   }
+
+  /** Extracts whether users provided the option to time travel a relation. */
+  private def getTimeTravelVersion(
```

Comment:
It's not really time travel in the case for streaming. Thinking about it more, all of this logic can live within DeltaOptions as well

Reply:
Yep. Stole this from a different module. Just haven't retitled it. I'll refactor it to DeltaOptions for both impacted modules https://github.com/delta-io/delta/blob/e365c2d3db3bf5bd9c70888fd704f43c98393837/src/main/scala/org/apache/spark/sql/delta/sources/DeltaDataSource.scala#L324